src/Aquifer.Data/Entities/HelpDocumentEntity.cs

```diff
@@ -0,0 +1,22 @@
+namespace Aquifer.Data.Entities;
+
+public class HelpDocumentEntity
+{
+    public int Id { get; set; }
+    public required HelpDocumentType Type { get; set; }
+    public required string Title { get; set; }
+    public required string Url { get; set; }
+    
+    [SqlDefaultValue("getutcdate()")]
+    public DateTime Created { get; set; }
```

Comment:
I'd put `= DateTime.UtcNow` on both of these as well.

Reply:
Why do we add this if the DB sets the value for us as defined already?

Reply:
The attribute is at the SQL server level and means I don't have to specify the datetime when inserting to the database directly with SQL. But if you instantiate the POCO it will still set a default value of minimum datetime which we would never want. EF may be smart enough to still use the default SQL value when it does the insert, but I think this makes it more explicit that when creating the entity the value has the current utcnow set and doesn't need to be added.